include/mlir/IR/OpImplementation.h

```diff
@@ -404,10 +404,25 @@ class OpAsmParser {
                                           ArrayRef<OperandType> arguments,
                                           ArrayRef<Type> argTypes) = 0;
 
-  /// Parse a region argument.  Region arguments define new values, so this also
-  /// checks if the values with the same name has not been defined yet.
+  /// Parse a region argument.  Region arguments define new values; so this also
+  /// checks if values with the same name have not been defined yet.
   virtual ParseResult parseRegionArgument(OperandType &argument) = 0;
 
+  /// Parse zero or more region arguments with a specified surrounding
+  /// delimiter, and an optional required argument count. Region arguments
+  /// define new values; so this also checks if values with the same names have
+  /// not been defined yet.
+  virtual ParseResult
+  parseRegionArgumentList(SmallVectorImpl<OperandType> &result,
+                          int requiredOperandCount = -1,
+                          Delimiter delimiter = Delimiter::None) = 0;
+  virtual ParseResult
+  parseRegionArgumentList(SmallVectorImpl<OperandType> &result,
+                          Delimiter delimiter = Delimiter::None) {
```

Comment:
Drop the default argument here, it won't be able to resolve `parseRegionArgumentList(result)` otherwise.

Reply:
Done, thanks.